Failed to build PyQt5-sip for py36 ERT installation

**Describe the bug**
ERT does not build for py36

```
 Failed to build PyQt5-sip
  Complete output (13 lines):
  running bdist_wheel
  running build
  running build_ext
  building 'PyQt5.sip' extension
  creating build
  creating build/temp.linux-x86_64-3.6
  gcc-10 -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/opt/hostedtoolcache/Python/3.6.15/x64/include/python3.6m -c apiversions.c -o build/temp.linux-x86_64-3.6/apiversions.o
  In file included from sipint.h:26,
                   from apiversions.c:24:
  sip.h:28:2: error: #error "This version of SIP requires Python v3.7 or later"
     28 | #error "This version of SIP requires Python v3.7 or later"
        |  ^~~~~
  error: command 'gcc-10' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for PyQt5-sip
```


**To Reproduce**
1. `pip install ert`

**Expected behavior**
Should build 


**Enviromment**
 - OS: github, various
 - ERT/Komodo Release: main
 - Python version: 3.6
 - Remote/HPC execution involved: no

**Additional context**
See https://github.com/equinor/ert/runs/6003988698?check_suite_focus=true

--- a/setup.py
+++ b/setup.py
@@ -93,6 +93,7 @@ setup(
         "psutil",
         "pydantic >= 1.9",
         "PyQt5",
+        "PyQt5-sip<=12.9.1; python_version <= '3.6'",
         "pyrsistent",
         "python-dateutil",
         "pyyaml",

